homeassistant/components/rflink/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "rflink",
   "name": "RFLink",
   "documentation": "https://www.home-assistant.io/integrations/rflink",
-  "requirements": ["rflink==0.0.58"],
+  "requirements": ["rflink==0.0.61"],
```

Comment:
0.0.61 contains an incompatible version spec for async_timeout:
https://github.com/aequitas/python-rflink/blob/182e10e8360dc133407b57efb5a25efa2c99cee5/setup.py#L60

The library should not limit the upper version of a requirement or put a limit that is compatible with other libraries and Home Assistant that use the same requirement.

Reply:
thanks. Issue opened on the library's repo